src/Components/benchmarkapps/BlazingPizza.Server/_Imports.razor

```diff
@@ -5,5 +5,6 @@
 @using Microsoft.AspNetCore.Components.Web
```

Comment:
Is it possible to move this benchmark to the aspnet/benchmarks repository? That's were we move all of them. It can still use the nightly aspnet version (latest builds actually) and you can still push your locally built assemblies (perf inner loop).

Reply:
> you can still push your locally built assemblies (perf inner loop)

Could you say more about how that works? What workflow or sequence of commands would the developer use? The scenario I'm focused on is:

1. I'm working inside the `aspnetcore` repo and have just made some changes to the internals of ASP.NET Core
2. I want to see what impact it has on the performance of `BlazingPizza.Server`, without pushing any changes to any repo

It's not clear to me how this would work if the benchmark was in a different repo, or how my locally-built ASP.NET Core binaries would get onto the agent.

Reply:
BTW I'm very happy to add a *copy* of this benchmark in aspnet/benchmarks. But it seems like it would be nice also to leave the original here in this repo to simplify inner-loop usage (unless I'm missing some good techniques).

Reply:
Resolving as the benchmarks aspect is now handled in https://github.com/aspnet/Benchmarks/pull/1804